### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by baezluise**
-Background images added.
-Menu section adapted to desktop.
-Contact form adapted to dekstop.
-Cards adapted with felxbox for desktop.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -427,3 +429,123 @@ form {
...
+  .contact-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    flex-direction: row;
+    background-image: url("contact-left-image.png");
+    background-repeat: no-repeat;
+    background-position: left;
+  }
```
Comment (zieeco) [inline styles.css:507]: - [x] Hey, great job with your design, however, let's kindly make a few adjustments to make our design better compared to the Figma design in the images below:
   - Let's align the form content to be on the same line with the card's content
   - Let's align the form button correctly as seen in the Figma design.
   - Let's improve the text also to be on the same line as the card and the text length as that in Figma. YOu can tweak the text-align, and play around with it to see what best fix as seen in the Figma design.




*your design*
Screenshot from 2023-01-27 22-21-54

*Figma design*
Screenshot from 2023-01-27 22-20-49

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @baezluise ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by baezluise** [comment]
-Contact form aligned.
-Button aligned.

File: styles.css (hunk 1 above)
Comment (Abenezer-Tilahun) [reply styles.css:507]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @baezluise 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git Hub-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**